src/effects/effectsmanager.cpp

```diff
@@ -70,81 +70,106 @@ const QSet<ChannelHandleAndGroup>& EffectsManager::registeredChannels() const {
     return m_pEffectChainManager->registeredChannels();
 }
 
-const QList<QString> EffectsManager::getAvailableEffects() const {
-    QList<QString> availableEffects;
+bool alphabetizeEffectNameIdPairs(const QPair<QString, QString>& pair1,
```

Comment:
QPair is bad, because you cannot see what .first and .second is.
It requires explanation on every usage. 
class NameIdPair {
  public: 
    QString Id;
    QString Name; 
}

would be self explainig